app/components/invitation-handler.js

```diff
@@ -1,32 +1,34 @@
 import {alias} from '@ember/object/computed';
 import {inject as service} from '@ember/service';
 import Component from '@ember/component';
+import {task, all, timeout} from 'ember-concurrency';
 
 export default Component.extend({
   invitation: null,
-
-  classNames: ['InvitationHandler'],
   session: service(),
+  router: service(),
   currentUser: alias('session.currentUser'),
 
   flashMessages: service(),
+
+  accept: task(function*() {
+    try {
+      const model = yield this.invitation.save();
+      yield all([timeout(500), model]);
```

Comment:
maybe centralize the 500ms wait into a meaningfully named constant? Something like:

```suggestion
      yield all([timeout(MIN_FEEDBACK_DELAY_MS), model]);
```

Naming TBD 😄